src/Microsoft.Sbom.Parsers.Spdx22SbomParser/Parser/SbomPackageParser.cs

```diff
@@ -109,7 +109,7 @@ private void ValidateSbomPackage(SPDXPackage sbomPackage)
             missingProps.Add(nameof(sbomPackage.LicenseConcluded));
         }
 
-        if (sbomPackage.LicenseInfoFromFiles == null || sbomPackage.LicenseInfoFromFiles.Count == 0)
+        if ((sbomPackage.LicenseInfoFromFiles == null && sbomPackage.FilesAnalyzed == true) || (sbomPackage.LicenseInfoFromFiles?.Count == 0 && sbomPackage.FilesAnalyzed == true))
```

Comment:
This condition can probably be simplified.  Maybe something like:

`if (sbomPackage.FilesAnalyzed && (sbomPackage.LicenseInfoFromFiles == null || bomPackage.LicenseInfoFromFiles?.Count == 0))`